frontend/src/components/ScreenSharingButton/ScreenSharingButton.tsx

```diff
@@ -64,6 +67,11 @@ const ScreenSharingButton = ({
                 <ScreenOff className="text-red-500" />
               )
             }
+            sx={{
+              marginTop: isOverflowButton ? '0px' : '4px',
+              marginLeft: isOverflowButton ? '12px' : '0px',
+            }}
```

Comment:
Just a reminder, the reason we handle this here instead of in the `ToolbarButton` component is because the three right panel buttons aren't aligned with the rest of the buttons.